tests/sensors/test_clean_up_reward.py

```diff
@@ -29,7 +29,7 @@
     }
 
 
-pytest.mark.skipif("Dexterity" not in holodeck.installed_packages(),
+@pytest.mark.skipif("Dexterity" not in holodeck.installed_packages(),
```

Comment:
Is there a reason why this one is `skipif` and line 36 is `skip`?

Reply:
Not a good reason! Great catch. They were both skipping so I assumed I had made it work.